src/procedures/the-cat-api/the-cat-api.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * The Cats Service provides access to the Cat API data from https://api.thecatapi.com/v1/.
+ *
+ * @service
```

Comment:
You might want to mark this as dev-only for the time being by adding `@alpha` ([reference](https://github.com/NetsBlox/services/blob/8c902b0ef85ec8025f121c55c3b6ff7045ed2d6b/src/procedures/nexrad-radar/nexrad-radar.js#L5-L7)), just cause we often avoid merging the first version of a service as a full release to editor.